weave/panels/panel_facet.py

```diff
@@ -14,6 +14,12 @@ class FacetDimsConfig:
     y: str
     select: str
     detail: str
+    xAxisLabel: weave.Node[str] = dataclasses.field(
```

Comment:
Curious if you thought about putting these settings inside `FacetConfig`. It seems like the pattern for `dims` are to be fields in the resulting table. But labels apply to the whole table.

Reply:
I felt like grouping them with x and y fields directly seemed intuitive to me. They appear next to the x and y expressions in the config and grouping the x y fields with the label made sense

Reply:
Not particularly opinionated about it tho

Reply:
I can see your point here. Another perspective is that `FacetDimsConfig` is the config for the facet _dimensions_... so when computing the facet data, we construct a table where each column is defined by the dimensions defined in the FacetDimsConfig. However, in this pattern, we do that for 4 of the fields, but not the labels. The labels are calculated across the entire input, as opposed to row-level. I'll approve and leave this up to you. Consider that if we break this pattern, what is left to differentiate the FacetDimsConfig from the FacetConfig? One could argue at that point, everything should just go into the top level config (unless you have another property that serves to logically separate the DimsConfig

Reply:
Once we commit to a config pattern, we must maintain support (or migrate) forever in the future - which i why i am being a bit careful here.